docs-src/python/how-to-define-a-signal-in-python.md

```diff
@@ -2,54 +2,38 @@
 id: how-to-define-a-signal-in-python
 title: How to define a Signal in Python
 sidebar_label: Define a Signal
-description: Define a Signal
-tags:
-  - developer-guide
-  - sdk
-  - python
+description: Use the `@workflow.signal` decorator to define a Signal.
 ---
 
```

Comment:
Maybe link to Signal concept page?

Reply:
It feels weird to just jump in here (unless this is part of a generated document where it doesn't come first). Maybe have some transition.